jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
Let's rebase this out. It's a weird commit, and even if you include it in the `.gitignore` later, the fact that it's at any point in the git history will override the `.gitignore`. See https://help.github.com/articles/removing-files-from-a-repository-s-history/